### ISSUE TITLE
[BUG] - [PF1E] - Players dropping items just clones default item pile and disappears whatever items they dropped

### ISSUE THREAD

**Issue by GaistH** [fantasycalendar/FoundryVTT-ItemPiles#411]
**Have you tried to reset Item Piles' system settings?**
No

**Describe the bug**
Players started dragging items to the ground to distribute loot to each other.  Instead, it just kept making more copies of the default item pile with its current inventory, while the items dragged off vanished.

**To Reproduce**

Take any actor, drag an item from their inventory onto the canvas, click yes to create a new pile.

**Expected behavior**
The dragged items creating new item piles.

**Screenshots**
![mKF2qpt](https://github.com/fantasycalendar/FoundryVTT-ItemPiles/assets/42808283/c247c91f-3958-4566-a63e-27868c0c0eb1)

**Setup:**

- Foundry Version: [11.306]
- System Version: [PF1E 9.1]
- Item Piles Version: [e.g. 2.7.3]
- Browser & version [e.g. Google Chrome 98.0.4758.82]

**Active modules:**

- Item Piles
- libwrapper
- socketlib
https://pastebin.com/zegaDxzc

**Comment by Haxxer** [comment]
Have you made your default item pile a linked actor? You shouldn't.

**Comment by GaistH** [comment]
It is an unlinked actor.

**Comment by Haxxer** [comment]
I can't reproduce on the latest version. I would recommend using Find The Culprit to find out of there's an offending module.

![image](https://github.com/fantasycalendar/FoundryVTT-ItemPiles/assets/12858387/346df128-4471-42ef-a852-711ca607f1e5)

**Comment by GaistH** [comment]
This seems to have been a residual issue with the migration from v10 to v11 Foundry for me -- I wound up with a duplicate of the default item pile and despite being unlinked it did also have the items pictured in the screenshot.  Thus it was spawning with those items while consuming the item dropped.  

When I deleted that actor and let item piles make a new one on prompting for dropping an item again, it worked OK again.  I also tried testing putting items back in that actor's inventory before dropping again but in this case it did not continue overwriting the items being dropped (They dropped alongside a copy of what was in the inventory.)

In the above screenshot it was after having to wrangle with the subordinate settings of the item pile while also having secondary currencies in the world; I had overlooked the option that splits out secondary currencies into regular coinage before we started the session and had adjusted those settings on the default item pile mid-session to try to prevent it; it was secondary currency in the screenshot.

I've not been able to reproduce it after re-creating the default item pile, so I suppose it's resolved; I seem to be having issues in general from the move to v11 due to things hitting max depth errors, even after the update to bump it up from a max depth of 15.
